website/docs/r/bigquery_dataset.html.markdown

```diff
@@ -73,6 +84,9 @@ The following arguments are supported:
 
 * `labels` - (Optional) A mapping of labels to assign to the resource.
 
+* `access` - (Optional) An array of objects that define dataset access for
```

Comment:
Can you add a "structure is documented below" statement and then docs for each of the attributes? search for that string in our existing docs and you'll find a bunch of examples.